net/release-notes/release-notes-2023/aspose-pdf-for-net-23-9-release-notes/_index.md

```diff
@@ -0,0 +1,112 @@
+|PDFNET-54555|Document does not allow to add text, throwing 'Invalid font name' Exception|Bug|
+|PDFNET-53938|Rectangles not being drawn on document|Bug|
+|PDFNET-45776|Printing from PDFViewer brings corrupt File or empty Page|Bug|
+|PDFNET-42780|A null referenced error on converting a PDF to PDF A3B|Bug|
+|PDFNET-42788|Closed stream error on saving a PDF to PDF A1B|Bug|
+|PDFNET-53167|PDF to PDF/A-1B: Slow conversion and significantly increased size|Bug|
+|PDFNET-54880|Regression: PDF to PDF/A_1B validation fails|Bug|
+|PDFNET-53635|After replacing text, image stamp is not applied|Bug|
+|PDFNET-53695|PDF to PDF/A: Misplaced text|Bug|
+|PDFNET-49490|PDF to PDF/A - output is not compliant due to usage of OTF fonts|Bug|
+|PDFNET-53307|PDF to PDF/A : Title mismatch|Bug|
+|PDFNET-50993|HTML to PDF: MS Mincho TTC font gets replaced|Bug|
+|PDFNET-42413|PDF to PDF/A - some gradient images are converted incorrectly|Bug|
+|PDFNET-53527|PDF to PDF/A-3A: Generated PDF_A_3A file cannot be redacted with Adobe|Bug|
+|PDFNET-53719|Regression: Text extraction overlaps text after extraction|Bug|
+|PDFNET-42802|Exception when joining 2 pdf files|Bug|
+|PDFNET-40958|PDF to PDF/A - text is corrupted and resultant file is not PDF/A compliant|Bug|
+|PDFNET-55195|Regression: Page.AddStamp() adds stamp to all pages|Bug|
+|PDFNET-55301|PDF to TXT: Text missing in output|Bug|
+|PDFNET-54912|There is a phenomenon where lines appear above and below the watermark|Bug|
+|PDFNET-54459|ArgumentOutOfRangeException occurs on using RedactionAnnotation.Redact()|Bug|
+|PDFNET-54955|System.NullReferenceException on PDF to DOCX conversion|Bug|
+|PDFNET-46919|PDF to TIF (Various parts of the document are disappearing)|Bug|
+|PDFNET-53709|PDFs will not save to XPS or print|Bug|
+|PDFNET-42007|PageIndex throws ArgumentException|Bug|
+|PDFNET-38754|PDF to PDFA2A/PDFA2B/PDFA3B: Converting PDF, generated from pptx using Aspose.Slides, to PDFA2A/PDFA2B/PDFA3B fails the compliance|Bug|
+|PDFNET-54065|[Aspose.PDF.Drawing] Squares instead of Japanese characters when converting PDF on Linux|Bug|
+|PDFNET-53610|The gradient is lost when converting PDF|Bug|
+|PDFNET-55334|System.NullReferenceException while accessing to the Field.Parent property in certain cases|Bug|
+|PDFNET-55350|Widget page index is reset to 0 after importing a fillable field|Bug|
+|PDFNET-51102|Parallel processing - code is not executing properly|Bug|
+|PDFNET-54927|Page.AddStamp() throws System.IndexOutOfRangeException|Bug|
+|PDFNET-50853|Aspose.PDF 21.10: After adding annotation there are no annotations in the resulting pdf file if open it in Google Chrome|Bug|
+|PDFNET-54333|When FlatteningAnnotations with PdfAnnotationEditor Throws System.IndexOutOfRangeException|Bug|
+|PDFNET-47851|PDF to TIFF - text is overlapped in output|Bug|
+|PDFNET-47306|Blank DOCX file generated|Bug|
+|PDFNET-50637|Redaction area is not correct|Bug|
+|PDFNET-50374|Aspose.Pdf 21.7: Cannot load Pcl on macOS|Bug|
```

Comment:
Issue PDFNET-50374 resolved with Aspose.PDF.Drawing (not Aspose.PDF for .NET). It should be removed from the list.